README.md

```diff
@@ -118,9 +118,16 @@ parameters, see below).  However, the author of this README
 strongly recommends to use Networkd instead of NetworkManager on
 servers and server-like systems.
 
-If the above example is sufficient you can install it via:
+If the above example is sufficient you can copy the drop-in config
+file in `/etc/dracut.conf.d/`:
 
     # cp example/20-wired.network  /etc/systemd/network
+    # cp example/networkd.conf /etc/dracut.conf.d
+
+Otherwise, you may add `systemd-networkd` files you want using the
+[`install_items`][items] or [`install_optional_items`][itemso] config
+options. Please note that at least one [`systemd.network`][networkfile] file
+is required for a working network.
```

Comment:
What about a more concise: '.. you can copy the example config files:'

Reply:
Since dracut reads the files under `/etc/dracut.conf.d` in lexicographic order perhaps it makes sense to rename `example/networkd.conf` to e.g. `90-networkd.conf`.

See also e.g. `/usr/lib/dracut/dracut.conf.d` for reference.

Reply:
Not sure what you mean by "more concise", I just wanted to add the sentence about needing to manually add `systemd.network` files when using networkd. I could remove it, if you think that's too verbose/too much hand holding.

Reply:
I think this is what you were thinking, let me know